bundles.md

```diff
@@ -0,0 +1,131 @@
+# Attestation Bundle
+in-toto attestations support multiple different predicate types with the idea that
+users will want to propagage multiple assertions about a single software package
+(Provenance, Code Review, Test Result, vuln scan, ...).
+
+Attestation Bundles provides a canonical method of storing those attestations in
+a single place.
+
+The attestations stored in a bundle may all come from the same source, signed by
```

Comment:
Should this move to the "data structure" section where you say what MAY happen?

Reply:
Sure, I also trimmed it down some.